sql/donut.sql

```diff
@@ -0,0 +1,48 @@
+DROP TABLE IF EXISTS users;
+DROP TABLE IF EXISTS members;
+
+CREATE TABLE members (
+    user_id int(11) NOT NULL AUTO_INCREMENT, -- Primary Key
```

Comment:
I think it only determines the number that is viewed in command line, but yeah 
I think I copied it a while ago from the Rudd dump, but I'll take it out. It is certainly misleading 
since the size of the integer types is fixed.